src/Compilers/Core/Portable/Syntax/SyntaxTriviaList.cs

```diff
@@ -65,7 +67,21 @@ public SyntaxTriviaList(IEnumerable<SyntaxTrivia>? trivias)
         {
         }
 
+        public static SyntaxTriviaList Create(ReadOnlySpan<SyntaxTrivia> trivias)
+        {
+            if (trivias.Length == 0)
+                return default;
+
+            if (trivias.Length == 1)
+                return new SyntaxTriviaList(trivias[0]);
+
+            return new SyntaxTriviaList(token: default, CreateNode(trivias), position: 0, index: 0);
+        }
+
         private static GreenNode? CreateNode(SyntaxTrivia[]? trivias)
+            => CreateNode(trivias.AsSpan());
```

Comment:
nit: seems like this is going from Array to Span to ReadOnlySpan, when we could just remove the call to AsSpan and use the implicit conversion straight from Array to ReadOnlySpan.

Reply:
oh.  let me change that.